### ISSUE TITLE
Use Alt Text of images in ActivityPub

### ISSUE THREAD

**Issue by H4kor** [H4kor/owl-blogs#27]
The alt text of images, which are automatically added to the ActivityPub object, are not transfered. Instead the file name is used. 

This should be changed to use the alt text if available, other leave the alt text empty.
